src/providers/wcs/qgswcsprovider.h

```diff
@@ -262,7 +262,7 @@ class QgsWcsProvider final: public QgsRasterDataProvider, QgsGdalProviderBase
     QString mTime;
 
     //! Specified bounding box
-    QString mBBOX;
+    QgsRectangle mBBOX;
```

Comment:
I think this needs a comment that the x/y coordinates in the rectangle may be flipped (if I'm reading things correctly!)

Reply:
Correct, comment added